### DISCUSSION TITLE
Vaultwarden for the win!

### DISCUSSION THREAD

**Discussion by Paulie420** [dani-garcia/vaultwarden#5196] [General]
w0w; I'm stuper impressed. I've been running a Bitwarden stack for ages and it worked well, but dang it had almost 10 containers - I switched over to Vaultwarden on a new docker/portainer server (putting all my eggs in one basket - last words???) and I'm stoked!

Vaultwarden spins right up, just modified my nginx proxy server to point to the new IP, and viola! SSL and all the fixins... 

Couple questions - I did NOT import my entire (old) /bwdata folder than contained my vault and many many backups of the vault - I just imported a .json export of the vault. Does that give me ALL the old Bitwarden data? Does Vaultwarden create backups like the old Bitwarden stack did? 

I'm excited to put Vaultwarden through its paces, but I'm happy with how little CPU it uses - not that Bitwarden was a dog or anything, but I do see less CPU with Vaultwarden!! BOOM!

**Comment by tessus** [comment]
Attachments are not exported. Thus I have no idea how to migrate those. But everything else should have been migrated by importing the json export.
